tests/test_mmdetectionmodel.py

```diff
@@ -239,6 +245,88 @@ def test_perform_inference_without_mask_output_with_automodel(self):
         # compare
         self.assertEqual(boxes[idx].astype(int).tolist(), [320, 323, 380, 365])
 
+    def test_perform_inference_with_text_prompt(self):
+        import nltk
+
+        from sahi.models.mmdet import MmdetDetectionModel
+
+        nltk.download("punkt_tab")
+        nltk.download("averaged_perceptron_tagger")
```

Comment:
is there a typo? @AMauri996 

```suggestion
        nltk.download("averaged_perceptron_tagger_eng")
```